src/components/cms/SkillCardScrollList/SkillCard.js

```diff
@@ -62,14 +62,19 @@ class SkillCard extends Component {
     };
   }
 
+  componentWillMount = () => {
```

Comment:
The use of this React lifecycle is now deprecated. Please try using some other approach.

Reply:
I hope this approach works fine.